tool/model/network.py

```diff
@@ -21,6 +19,36 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass(frozen=True)
+class NetworkDescription:
+    """Top-level elements that describe a network."""
+    id: str
+    name: str
+
+    @classmethod
+    def from_network_object(cls, network: Dict[str, Any]) -> "NetworkDescription":
+        return cls(
+            id=network["id"],
+            name=network.get("name", f"Unnamed Network <{network['id']}>")
+        )
+
+    def to_network_object(self) -> Dict[str, Any]:
+        return {
+            "id": self.id,
+            "name": self.name,
+            "crs": {
+                "name": "urn:ogc:def:crs:OGC::CRS84",
+                "uri": "http://www.opengis.net/def/crs/OGC/1.3/CRS84"
+            },
+            "links": [
+                {
+                    "rel": "describedby",
+                    "href": "https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0__2__0/schema/network-schema.json"
```

Comment:
Realised that I can just suggest the necessary change

```suggestion
                    "href": "https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0__3__0/schema/network-schema.json"
```